runtime/mux.go

```diff
@@ -229,6 +229,9 @@ func WithHealthEndpointAt(healthCheckClient grpc_health_v1.HealthClient, endpoin
 					return
 				}
 
+				w.Header().Set("Content-Type", "application/json")
+				w.Header().Set("Grpc-Metadata-Content-Type", "application/grpc")
```

Comment:
Could we add a comment to explain why we need both of these please?

Reply:
The gRPC RFC indicates that the header should be [application/grpc](https://datatracker.ietf.org/doc/html/draft-kumar-rtgwg-grpc-protocol-00). However when I call the other endpoints I get `application/json` and `Grpc-Metadata-Content-Type` `application/grpc`. So to being consistent with other endpoints I simply copied the headers received.

What do you want as commentary? The fact that i have to manually set the headers or should I talk about the RFC instead?
Thanks

Reply:
I don't think we need to include the translated grpc content type header tbh. It's there normally because we automatically translate those headers, but we don't need to set it here explicitly. If we remove it, there's little reason for a comment at all, I think it's pretty self explanatory why we need to set the JSON content type.

Reply:
Thanks for your feedback, I have removed the grpc header and then no comments are needed